src/main/client/src/utils/layers.js

```diff
@@ -76,3 +76,25 @@ export const addLayerToUserGroup = (
 export const layerViewable = (layer: Object, mapScale: number) => (
     layer.maxScale < mapScale && (layer.minScale > mapScale || layer.minScale === 0)
 );
+
+/**
+ * Check if layer is contract layer
+ *
+ * @param {Object} layer to determinate if it is contract layer.
+ * @returns {boolean} Is layer contract layer.
+ */
+export const isContract = (layer: Object) => layer && layer.type === 'agfl'
+    && ((layer.relations.find(r => r).relationColumnIn === null
+        && layer.relations.find(r => r).relationColumnOut === null)
+        || layer.relations.find(r => r).relationLayerId === null
+        || layer.name.toLowerCase() === 'tlaite sopimushallinta') === true;
+
```

Comment:
- Miksi tässä tarkastetaan että onko palautusarvo true? tuossa on käytönnössä kirjoitettu if (true === true)
  - Metodin jsdocissa voisi myös mainita, että etsitään tarkoituksella vain yksi relaatio

Vaikka logiikka onkin kopioitu suoraan alkuperäisestä tarkistuksesta, olisiko kuitenkin varmempaa tarkastaa että arvot ovat olemassa ja ei kaadu, kun tuo layer.relations.find() voi käytännössä palauttaa undefined? tyyliin:
```suggestion

export const isContract = (layer: Object) => {
    if (layer && layer.name && layer.type && layer.relations) {
        const relation = layer.relations.find(r => r);

        if (relation && layer.type === 'agfl') {
            return (relation.relationColumnIn === null
                && relation.relationColumnOut === null)
                || relation.relationLayerId === null
                || layer.name.toLowerCase() === 'tlaite sopimushallinta';
        }

        return false;
    }

    return false;
};

```

Reply:
muutettu esimerkin mukaiseksi